### PR TITLE
refactor: Move remote device service ssh tunneling artifacts to new example repo

### PR DESCRIPTION
**Pull request by jim-wang-intel**
Move all the artifacts for SSH Tunneling examples of remote device service from
 edgexfoundry-holding/external-device-security-examples/pull/1 into this PR

 Closes: #5

Signed-off-by: Jim Wang <yutsung.jim.wang@intel.com>

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [X] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [X] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #5 

## What is the new behavior?
Add artifacts of reference example of ssh tunneling for device service.

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

## Other information
Relate to: edgexfoundry/edgex-docs/issues/134

### CODE REVIEW

File: security/remote_devices/ssh-tunneling/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Reference Implementaton for SSH Tunneling on EdgeX Device Services
+
+This is a reference implementation for using SSH Tunneling on EdgeX device-virtual services.
+
+## Build and Run
+
+Please see the detailed steps in "how-to guide" for SSH Tunneling in documentation repository here: <https://github.com/edgexfoundry/edgex-docs/blob/master/docs_src/microservices/security/Ch-SSH-Tunneling-HowToSecureDeviceServices.md#put-it-all-together>.
```
Comment (jim-wang-intel) [inline security/remote_devices/ssh-tunneling/README.md:7]: Note: this is the future hyperlink after merging into master of that repo.  Currently please use [this URL for reference](https://github.com/jim-wang-intel/edgex-docs/blob/device-ssh-tunneling-how-to/docs_src/microservices/security/Ch-SSH-Tunneling-HowToSecureDeviceServices.md#put-it-all-together)

File: security/remote_devices/ssh-tunneling/build.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+#  SPDX-License-Identifier: Apache-2.0'
+#  ----------------------------------------------------------------------------------
+
+set -e
+
+DEFAULT_SSH_KEY="ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDKvsFf5HocBOBWXdVJKfQzkhf0K8lSLjZn9PX84VdhHyP8n1mzfpZywA4vsz8+A3OsGHAr2xpkyzOS0YkwD7nrI3q1x0A0+ANhQNOaKbnfQRepTAES3FPm5n0AbNVfgOre3RR2NLOt6M5m3mA/MERNer1fEp6BM96sdU0o3KjqwFGkPufoQrVkpz2691MZ6/ACDc+lk7uQrinsB4YxM7ctiLNl4I1A3TJgVv0jkJImUCHaThYj3XoaqUqUjQFTS7SlFfkXuk13EjNfRzqPwKFnVvGTUaYzaBV5S4wt5XCxhLfs497M2k5zmNx3HFY/GEyeoroCpjsiXkm+HcgdIYb7 root"
```
Comment (bnevis-i) [inline security/remote_devices/ssh-tunneling/build.sh:22]: Suggest corrupt this key so that this doesn't get used in the wild....

Comment (jim-wang-intel) [reply security/remote_devices/ssh-tunneling/build.sh:22]: ok, will do

**Comment by jim-wang-intel** [comment]
@lenny-intel, @jpwhitemn , @tonyespy 
Note: the current example org. structure is `security/remote_devices` from the base.  Let me know if it has to change to something else.

**Comment by bnevis-i** [comment]
@jim-wang-intel Ready to merge this?

**Pull request merged**